content/sensu-core/2.0/reference/entities.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Entities"
+description: "The entities reference guide."
+weight: 1
+version: "2.0"
+product: "Sensu Core"
+platformContent: true
+menu:
+  sensu-core-2.0:
+    parent: reference
+---
+
+# Entities
+
+### Why "Entities"?
+
+Sensu 2.0 uses an updated data model to address the latest paradigm shift in infrastructure and software delivery like containers and "serverless". Sensu is no longer confined to the concept of a client, we have "entities" that can can represent anything.
+
+### What is a Sensu entity?
+
+Entities, formally known as clients in Sensu 1.x, are monitoring agents (e.g. server, container, etc) that need to be monitored. They contain system information such as the hostname, OS, platform, and version. Entities not only provide context to event data but an event's uniqueness is determined by the check name and the ID of the entity upon which the check ran.
```

Comment:
The first sentence is hard to parse for me. I think I don't get "monitoring agents (e.g. server, container, etc) that need to be monitored." Maybe removing the server/container detail or mentioning whether each entity is a device monitored or it implies a running agent.

Reply:
I also think speaking in the singular under this section would be nice to clarify (vs plural everywhere else).